documentation/book/con-cluster-operator.adoc

```diff
@@ -0,0 +1,21 @@
+[id='cluster-operator-{context}']
+= Overview of Cluster Operator
+
+The Cluster Operator is in charge of deploying a Kafka cluster alongside a Zookeeper ensemble. As part of the Kafka cluster, it can also deploy the Topic Operator which provides operator-style topic management via ConfigMaps. 
+
+ifdef::Kubernetes[]
+The Cluster Operator can also deploy a Kafka Connect cluster which connects to an existing Kafka cluster. 
+endif::Kubernetes[]
+
+On OpenShift, you can create a cluster and connect to an existing Kafka cluster using the Source2Image feature, which provides an easy way of including more connectors.
+
+.Example Architecture diagram of the Cluster Operator.
+image::cluster_operator.png[Cluster Operator]
+
+When the Cluster Operator is up, it _watches_ for certain {ProductPlatformName} resources containing the desired Kafka or Kafka Connect cluster configuration. A `Kafka` resource is used for the Kafka cluster configuration, and a `KafkaConnect` resource is used for Kafka Connect cluster configuration.
```

Comment:
The is another resource `KafkaConnectS2I` for the Soruce2Image enabled Kafka Connect. I think it should be listed here as well.